dask_searchcv/model_selection.py

```diff
@@ -769,7 +776,26 @@ def fit(self, X, y=None, groups=None, **fit_params):
             Parameters passed to the ``fit`` method of the estimator
         """
         estimator = self.estimator
-        self.scorer_ = check_scoring(estimator, scoring=self.scoring)
+        # self.scorer_ = check_scoring(estimator, scoring=self.scoring)
+        if _HAS_MULTIPLE_METRICS:
+            from sklearn.metrics.scorer import _check_multimetric_scoring
+
+            self.scorer_, self.multimetric_ = _check_multimetric_scoring(
+                estimator, scoring=self.scoring)
+        else:
+            self.scorer_ = check_scoring(estimator, scoring=self.scoring)
+
+        # This is {None, True, False}
+        # None indicates scikit-learn < 0.19 (when multimetric was added)
+        # True indicates multiple metrics were used
+        # False indicates a single metric was used
+        multimetric = getattr(self, 'multimetric_', None)
+        if multimetric:
+            multimetric = self.scorer_.keys()
+        elif multimetric is False:
+            # sklearn >= 0.19.0, but single metric
+            self.scorer_ = self.scorer_['score']
```

Comment:
I think this is slightly cleaner/clearer:

```python
if _HAS_MULTIPLE_METRICS:
    from sklearn.metrics.scorer import _check_multimetric_scoring
    
    scorer, multimetric = _check_multimetric_scoring(estimator, scoring=self.scoring)
    if not multimetric:
        scorer = scorer['score']
    # Attribute only on sklearn >= 0.19.0
    self.multimetric_ = multimetric
else:
    scorer = check_scoring(estimator, scoring=scoring)
    multimetric = False

self.scorer_ = scorer
```

Then pass the boolean `multimetric` to `build_graph`, where the list of score names will be made if needed. I think this keeps the necessary logic localized, and also avoids reassigning variables with a conceptually different value.